tools/bin/check_images_exist.sh

```diff
@@ -82,16 +82,18 @@ checkPlatformImages() {
 
 checkNormalizationImages() {
   echo -e "$blue_text""Checking Normalization images exist...""$default_text"
-  # the only way to know what version of normalization the platform is using is looking in NormalizationRunnerFactory.
   local image_version;
-  factory_path=airbyte-commons-worker/src/main/java/io/airbyte/workers/normalization/NormalizationRunnerFactory.java
+  definition_file_path=airbyte-config/init/src/main/resources/seed/destination_definitions.yaml
   # -f True if file exists and is a regular file
-  if ! test -f $factory_path; then
-    echo -e "$red_text""No NormalizationRunnerFactory found at path! H4LP!!!""$default_text"
+  if ! test -f $definition_file_path; then
+    echo -e "$red_text""Destination definition file not found at path! H4LP!!!""$default_text"
   fi
-  image_version=$(cat $factory_path | grep 'NORMALIZATION_VERSION =' | cut -d"=" -f2 | sed 's:;::' | sed -e 's:"::g' | sed -e 's:[[:space:]]::g')
-  echo -e "$blue_text""Checking normalization images with version $image_version exist...""$default_text"
-  VERSION=$image_version 
+  normalization_image_versions=$(cat $definition_file_path | grep 'normalizationTag:' | cut -d":" -f2 | sed 's:;::' | sed -e 's:"::g' | sed -e 's:[[:space:]]::g')
```

Comment:
how hard would it be to install `yq` instead of doing this? (nonblocking)

Reply:
I don't think it will take long. But I suggest leaving it as it is. I plan to return to this file anyway, at the end of the platform separation and normalization work.